package.json

```diff
@@ -50,5 +50,8 @@
     "rollup-plugin-node-resolve": "^3.3.0",
     "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-uglify": "^4.0.0"
+  },
+  "dependencies": {
+    "@volusion/element-components": "^1.0.6"
```

Comment:
To support this feature, we need to get the names of all available components, and `element-components` seems like the logical place to get that from.

But, including `element-components` here makes our dependency structure even more complicated, so I am a little concerned about that. Setting it as an external dependency should help, but I'm wondering if there are other ways to handle this.